### PR TITLE
Add/semantic verbal fluency task

### PR DESCRIPTION
**Pull request by JuneOh710**

### CODE REVIEW

File: docs/_library/affective-conflict-resolution-task.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+layout: experiment
+name:  "affective-conflict-resolution-task"
+maintainer: "@JuneOh710"
+github: "https://github.com/JuneOh710/affective-conflict-resolution-task"
+preview: "https://juneoh710.github.io/affective-conflict-resolution-task/"
+tags:
+- test
+- experiment
+---
+This was a legacy experiment that has been ported into its Experiment Factory Reproducible Container version. If you'd like to make the experiment, it's documentation, or use better, please contribute at the repositories
```
Comment (vsoch) [inline docs/_library/affective-conflict-resolution-task.md]: Do you want to change the descriptions this time to better reflect the experiments? I don't think it's correct that this was a legacy expfactory experiment.

**Comment by vsoch** [comment]
Also you need to rebase with the master branch here, and not the fork of your master branch. I did it for you the other times - that's why the CI is failing.

File: docs/_library/semantic-verbal-fluency-task.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+name:  "semantic-verbal-fluency-task"
+maintainer: "@JuneOh710"
+github: "https://github.com/JuneOh710/semantic-verbal-fluency-task"
+preview: "https://juneoh710.github.io/semantic-verbal-fluency-task/"
+tags:
+- test
+- experiment
+---
+
+This was a legacy experiment that has been ported into its Experiment Factory Reproducible Container version. If you'd like to make the experiment, it's documentation, or use better, please contribute at the repositories
```
Comment (vsoch) [inline docs/_library/semantic-verbal-fluency-task.md]:
```suggestion
```

**Comment by vsoch** [comment]
@JuneOh710 please fix your fork so that it doesn't bring in new commits - this is the last PR I'm going to offer to fix for you. We should not see all those extra (old) commits that have been squashed. I really appreciate your contributions but it's not fair that you create a lot of extra work for me every time. It will be good practice for you to open PRs properly because not all maintainers will be willing to help. Again, thank you!

File: docs/_library/semantic-verbal-fluency-task.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+---
+layout: experiment
+name:  "semantic-verbal-fluency-task"
+maintainer: "@JuneOh710"
+github: "https://github.com/JuneOh710/semantic-verbal-fluency-task"
```
Comment (vsoch) [inline docs/_library/semantic-verbal-fluency-task.md:5]: This repository does not exist.
```suggestion
github: "https://github.com/JuneOh710/semantic-verbal-fluency"
```
And you've already added that repository https://github.com/expfactory/experiments/blob/master/docs/_library/semantic-verbal-fluency.md.

**Comment by danieljwilson** [comment]
Apologies for all the back and forth on this task - it is part of a battery I am hoping to launch soon - I was wondering if there is anything I/we can do to have this appear in the library? Right now I can access the experiment at [this link](https://github.com/expfactory/experiments/blob/master/docs/_library/semantic-verbal-fluency.md) but I do not see it when I search for it in the library?

image

**Comment by vsoch** [comment]
It was a bug - the markdown was missing the top three lines for the frontend matter. Try again - should be there now!

**Comment by danieljwilson** [comment]
It's there now, great! Thank you!
🙏

**Comment by vsoch** [comment]
Sure thing! Okay I'm back to bed, will be awake again for work in a few hours :)

**Pull request closed without merging**